modes/basic-dev-mode/src/index.js

```diff
@@ -89,39 +89,8 @@ function modeFactory({ modeConfiguration }) {
         // disabled
       };
 
-      const toolGroupId = 'default';
-      toolGroupService.createToolGroupAndAddTools(toolGroupId, tools);
-
-      let unsubscribe;
-
-      const activateTool = () => {
-        toolbarService.recordInteraction({
-          groupId: 'WindowLevel',
-          interactionType: 'tool',
-          commands: [
-            {
-              commandName: 'setToolActive',
-              commandOptions: {
-                toolName: 'WindowLevel',
-              },
-              context: 'CORNERSTONE',
-            },
-          ],
-        });
-
-        // We don't need to reset the active tool whenever a viewport is getting
-        // added to the toolGroup.
-        unsubscribe();
-      };
-
-      // Since we only have one viewport for the basic cs3d mode and it has
-      // only one hanging protocol, we can just use the first viewport
-      ({ unsubscribe } = toolGroupService.subscribe(
-        toolGroupService.EVENTS.VIEWPORT_ADDED,
-        activateTool
-      ));
+      toolGroupService.createToolGroupAndAddTools('default', tools);
```

Comment:
Wow - I really like this change.

Reply:
Is there some way you could register the tool sets with the customization service - will think about that, and see if we can come up with a way to have the extensions define default sets of tools by registering pieces with the customization and toolbar services, but I don't htink it belongs in this PR, jsut something to think about.